.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: trusty
 language: node_js
 
 node_js:
-  - "7.10.0"
+  - "8.0.0"
```

Comment:
why not use latest version "9.3.0" ?

Reply:
@orenyodfat 

> why not use latest version "9.3.0" ?

I think we should set this to the minimum version that we require for building and testing arc-js.

Reply:
On travis I think we should use the lowest common denominator version for all users, to ensure we are running these tests on their behalf and not silently leaving them behind.

Reply:
Usually I tend to take @dkent600 side and try to work with the minimum.
But on the blockchain space, where everything is moving so fast I think we should try to use updated tools. As things might break more easily.

Reply:
are you plan to address that on this pr ?

Reply:
@orenyodfat  I didn't see Adam's comment.  I'll go ahead and bump up the version

Reply:
this has been resolved

Reply:
this has been resolved

Reply:
@tibetsprague wird ..the version is still 8.0.0 ..or i am missing something

Reply:
I'm fixing it now

Reply:
oops, sorry that was the wrong place for that comment :)